Library/ViewModels/CommentsViewModel.swift

```diff
@@ -63,15 +62,26 @@ public final class CommentsViewModel: CommentsViewModelType,
     }
     .observeValues { print($0) }
 
-    self.avatarURL = currentUser.map { URL(string: $0.avatar.medium) }
+    self.configureCommentComposerViewWithData = Signal
+      .combineLatest(self.projectProperty.signal.skipNil(), currentUser.signal)
+      .takeWhen(self.viewDidLoadProperty.signal)
+      .map { project, currentUser in
+        let isBacker = userIsBackingProject(project)
+
+        guard let user = currentUser else {
+          return (nil, isBacker)
+        }
 
-    // When Project is supplied to the config, we will use that to determine who can comment on the project
-    // and also to determine whethere input area is visible.
-    self.inputAreaVisible = self.viewDidLoadProperty.signal.mapConst(true)
+        return (URL(string: user.avatar.medium), isBacker)
```

Comment:
Just a tiny thing, usually prefer to create the object in a separate constant value before passing into the tuple.
Maybe, 
```
let url = URL(string: user.avatar.medium)
return (url, isBacker)
```

Reply:
for readability

Reply:
Updated.